core/src/main/resources/lib/layout/html.jelly

```diff
@@ -0,0 +1,177 @@
+<!--
```

Comment:
@jglick yes, copied from layout.jelly. I suppose one could add an option to the existing layout.jelly. Is this important to refactor?